src/core/public/workspace/workspaces_client.ts

```diff
@@ -70,6 +70,8 @@ export class WorkspacesClient {
         }
       }
     );
+
+    this.init();
```

Comment:
I originally moved the calling of `init()` to workspace plugin `setup()` function so it only calls `init()` when workspace is enabled. Any reason to move it to `constructor()`?